modules/benchmark/R/align_by_obeservation_one.R

```diff
@@ -0,0 +1,47 @@
+#################################################################
+#'
+#' @title align_by_observation_one
+#' @param observation_one a vector of plant fucntional types, or species
+#' @param observation_two anouther vector of plant fucntional types, or species
+#' @param custom_table a table that either maps two pft's to one anouther or maps custom species codes to bety id codes. 
+#' In the second case, must be passable to match_species_id. 
+#' @return \code{vector} Returns a vector of PFT's/species from observation_one that matches the order of observation_two } 
+#' 
+#' @author Tempest McCabe
+#' @examples
+#' 
+#' observation_one<-c("AMCA3","AMCA3","AMCA3","AMCA3")
+#' observation_two<-c("a", "b", "a", "a")
+#' table<-list()
+#' table$plant_functional_type_one<- c("AMCA3","AMCA3","ARHY", "ARHY")
+#' table$plant_functional_type_two<- c('a','a','b', 'b') # PFT groupings
+#' table$input_code<-c("AMCA3","AMCA3","ARHY", "ARHY") # Species
+#' table<-as.data.frame(table)
+#'
+#' format_one<-"species_USDA_symbol"
+#' format_two<-"plant_funtional_type"
```

Comment:
Yes they are. They're used for the checks of the format, which tells align_pft if it should be using custom_table as somethign to pass to match_species_id or treat it as a plant_functional_type. 


Based on your comments though, it think I should restructure + expand on my examples.

Reply:
@mccabete look at the arguments you are passing in to this function carefully -- ```format_one``` and ```format_two``` are not, in fact, passed into the function, nor are they used internally in the function or in a step that you do before you call the function. Since these variables are not used they should be deleted from the example, since their presence is confusing/misleading.

Reply:
Ah, sorry, I got confused by my own example.